packages/docs/src/page-configs/ui-elements/switch/examples/Error.vue

```diff
@@ -1,5 +1,5 @@
 <template>
```

Comment:
yarn.lock doesn't seem to need any change.
So keeping it the way it was would make sense from my perspective.